include/kodlab_mjbots_sdk/mjbots_robot_interface.h

```diff
@@ -30,32 +30,29 @@ struct RealtimeParams {
   int lcm_cpu = 0;
 };
 
-class MjbotsRobotInterface {
+/*!
+ * @brief A class that allows interaction with the Mjbots Moteus Motor Controllers
+ * 
+ */
+class MjbotsRobotInterface  {
```

Comment:
Lets rename this class to `MjbotsHardwareInterface`

Reply:
Renamed in ea7778a442081ea4ef04a3c9f17389b0c9217d31.